imgui_plot.h

```diff
@@ -1075,6 +1076,8 @@ namespace ImGui
     struct ImAreaProperties
     {
         ImU32 area_fill_col = 0;
+        ImU32 area_hover_col = 0;
+        const char* tooltip = NULL;
```

Comment:
I would recommend placing the "tooltip" labels information in the arguments of the `DrawArea` as an additional parameter (i.e., `const char* series` as done in `DrawMarkers`).

Reply:
What is the reason behind this choice?
Since there is one tooltip per area/single series, I thought it was better to make it a property.

Every call to `DrawSomething` only takes care of one area (one series) at the time. Could providing info about other areas/series introduce confusion?

Reply:
The reason is that the properties define the features of the plot that do not change with the data.  The goal is to allow the user to define the properties structure once and reuse it in all instantiations of ImArea.  This would mean that the data specific information (i.e., the series labels) would be given dynamically in each call to DrawArea.

Does that reasoning make sense?

Reply:
Yes, I see, thanks for explaining.
I'll update the code with the correct workflow